chunkenc/bstream.go

```diff
@@ -39,6 +39,8 @@
 // OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+//lint:file-ignore U1000 Ignore all unused code.
```

Comment:
why is this needed?

Reply:
This file has a few functions that aren't used anywhere but I wasn't sure if we wanted to remove them or not.

```
chunkenc/bstream.go:56:6: func newBWriter is unused (U1000)
chunkenc/bstream.go:60:19: func (*bstream).clone is unused (U1000)
```